File: src/main/java/dev/yurii/vk/polygon/vkauth/VkApiConfiguration.java
package dev.yurii.vk.polygon.vkauth;

import com.vk.api.sdk.client.VkApiClient;
import com.vk.api.sdk.httpclient.HttpTransportClient;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class VkApiConfiguration {

    @Bean
    public VkApiClient vkApiClient() {
        return new VkApiClient(new HttpTransportClient());
    }
}

File: src/main/java/dev/yurii/vk/polygon/vkoauth2/VKApiConfiguration.kt
package dev.yurii.vk.polygon.vkoauth2

import com.vk.api.sdk.client.VkApiClient
import com.vk.api.sdk.httpclient.HttpTransportClient
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
open class VKApiConfiguration {

    @Bean
    open fun vkApi(): VkApiClient {
        return VkApiClient(HttpTransportClient())
    }
}
